scalafix-tests/output/src/main/scala/test/explicitResultTypes/Rename.scala

```diff
@@ -1,11 +1,11 @@
 
 package test.explicitResultTypes
 
-import scala.collection.{Seq => SSeq}
+import scala.collection.immutable.{List => LList}
 import java.lang.{Boolean => JBoolean}
 
 object Rename {
-  type Seq = Int
-  def foo(a: Int*): SSeq[Int] = identity(a)
+  type List = Int
+  def foo(a: Int): LList[Int] = List(a)
```

Comment:
I haven't looked at the git history, but maybe there was something with varargs/identity that this was specifically testing? In that case, maybe it's better to go for:
```suggestion
  def foo(a: Int*): LList[Int] = identity(a.toList)
```